src/clojure/clj_ldap/client.clj

```diff
@@ -43,6 +44,20 @@
 (defn encode [attr]
   (.getValue attr))
 
+(defn ssl-protocol-mapping
+  "Converts user protocol settings into valid string constants"
+  [ssl-protocols]
+  (let [valid-protocols #{"TLSv1.3" "TLSv1.2" "TLSv1.1" "TLSv1"}
+        protocol-set (set ssl-protocols)
+        protocol-diff (difference protocol-set valid-protocols)]
+    (when not-nil? protocol-diff
+      log/info (trs "Unsuported value {0} passed into SSL protocol" protocol-diff))
```

Comment:
Could use suggestions on a prettier way to compare the values and subsequently remove unsupported value from list?

Reply:
```
(remove #(contains? valid-protocols %) ssl-protocols))
```
https://clojuredocs.org/clojure.core/remove
https://clojuredocs.org/clojure.core/contains_q 
https://clojuredocs.org/clojure.core/some_q

Will return any entries that aren't mapped, or an empty sequence.

Reply:
thank you!